src/wagtail_bynder/models.py

```diff
@@ -333,8 +375,6 @@ def update_from_asset_data(
         self,
         asset_data: dict[str, Any],
     ) -> None:
-        super().update_from_asset_data(asset_data)
-
         primary_derivative_name = getattr(
             settings, "BYNDER_VIDEO_PRIMARY_DERIVATIVE_NAME", "Web-Primary"
```

Comment:
```suggestion
            settings, "BYNDER_VIDEO_PRIMARY_DERIVATIVE_NAME", "WebPrimary"
```

to match the README and the tests settings